dot/sync/chain_sync.go

```diff
@@ -492,7 +506,6 @@ func (cs *chainSync) setMode(mode chainSyncState) {
 // it would be better to have some sort of standard deviation calculation and discard any outliers (#1861)
 func (cs *chainSync) getTarget() *big.Int {
```

Comment:
can you add a test case for this function specifically?

Reply:
@noot There is an existing test case here for this method: https://github.com/ChainSafe/gossamer/blob/e69968a7adacd7952b396b1296c3877b0cbb7288/dot/sync/chain_sync_test.go#L201

Is it not enough?

Reply:
not really, because there's no outliers it's removing

Reply:
Updated ✔